### PR TITLE
Spelling errors and simple grammar

### PR DESCRIPTION
**Pull request by SwiftSergio**
# Spelling errors and simple grammar fixes

I originally forked the project to find something more substantial to contribute to, then I came across some of these errors and fixed them while I continued searching for other issues. However, as this is my first pull request I opted to keep things simple!

Also It doesn't seem like my proposed changes lines up with any current open issues so I did not link to anything.

## What I changed

- Simple grammar fixes (such as missing words, or capitalization - please see my question about this below)
- Misspelled words
- Missing whitespaces to remove warnings from SwiftLint

## Questions moving forward

Some of the comments due not follow any strict format. For example at times it will seem like a complete grammatically sound sentence and at other times it will be written just a side note with no concern for grammar. I assume this is not a concern?

## Feedback

As I said this is my first PR so any feedback you have would be greatly appreciated! Also thank you for creating an environment that facilitates this effort!

### CODE REVIEW

File: Unwrap/Activities/Challenges/ChallengesDataSource.swift [hunk 1]
```diff
@@ -73,16 +73,17 @@ class ChallengesDataSource: NSObject, UITableViewDataSource {
             return cell
         }
     }
+    
     /// Swipe to delete
     func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
-        if indexPath.section == 1 && User.current.dailyChallenges.count > 0{
+        if indexPath.section == 1 && User.current.dailyChallenges.count > 0 { //** Added spaaces to silence warnings from swift lint **
```
Comment (Carrione) [inline Unwrap/Activities/Challenges/ChallengesDataSource.swift:79]: No need to add a comment 😃

File: Unwrap/Activities/Home/HomeCoordinator.swift [hunk 2]
```diff
@@ -34,7 +34,8 @@ class HomeCoordinator: Coordinator, AlertShowing {
         if isFirstRun {
             /// If this is the first time the app is running, wait a tiny fraction of time before showing the welcome screen.
             DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
-                // Put the contents of showTour in here directly avoid trying to capture `self` during an initializer.
+                // Put the contents of showTour in here directly to avoid trying to capture `self` during an initializer.
+                // Added to 'to' between 'directly' and 'avoid'
```
Comment (Carrione) [inline Unwrap/Activities/Home/HomeCoordinator.swift:38]: No need to add this comment. Changes are visible.

File: Unwrap/Activities/Welcome/TourHostViewController.swift [hunk 3]
```diff
@@ -11,7 +11,8 @@ import UIKit
 class TourHostViewController: UIViewController {
     @IBOutlet var pageControl: UIPageControl!
 
-    /// Configures our child so that it can report back how many pages if contains and when page changes happen so we can update the page control.
+    // ** CHANGED 'IF' to 'IT' **
```
Comment (Carrione) [inline Unwrap/Activities/Welcome/TourHostViewController.swift:14]: No need to add this comment. Changes are visible thanks to git.

**Review by Carrione — COMMENTED** [review]
Hi Sergio, I've looked at your PR. Please take a look at my comments. I also see there is a merge conflict in the Unwrap/Activities/Challenges/ChallengesDataSource.swift file , can you take a look at it please.

**Review by SwiftSergio — COMMENTED** [review]
Removed unnecessary comments and conflict should be resolved.

**Comment by Carrione** [comment]
Don't forget the Unwrap/Activities/Home/HomeCoordinator.swift file 😄

**Review by Carrione — COMMENTED** [review]
Thank you 😄

**Comment by SwiftSergio** [comment]
I Overlooked that! 😅 Thank you.

**Review by Carrione — COMMENTED** [review]
Thank you 👍🏻

**Comment by twostraws** [comment]
I love these changes – they really help make the code easier to read. Thank you very much! 🙌

**Pull request merged**